news/tests.py

```diff
@@ -100,7 +102,10 @@ def setUp(self):
         self.event = Event.objects.create(
             title='FUTURE',
             image=SimpleUploadedFile(name='img.jpg', content=simple_jpg, content_type='image/jpeg'),
+            number_of_tickets=40,
         )
+        self.timeplace = TimePlace.objects.create(event=self.event, start_time=timezone.localtime() + timedelta(minutes=5),
+                                      end_time=timezone.localtime() + timedelta(minutes=10), number_of_tickets=29)
```

Comment:
I think wrapping the code like this is more readable: 😊 
```suggestion
        self.timeplace = TimePlace.objects.create(
            event=self.event, number_of_tickets=29,
            start_time=timezone.localtime() + timedelta(minutes=5),
            end_time=timezone.localtime() + timedelta(minutes=10),
        )
```